app/views/search/_collection_search_results.html.erb

```diff
@@ -1,40 +1,36 @@
 <div class="pagination-summary">
-  <%= page_entries_info @collections, entry_name: 'record' %>
+  <%= page_entries_info collections, entry_name: 'record' %>
```

Comment:
Seems like we would want this to say 'No collections found' instead of 'No records found'.